mRemoteV1/Resources/Help/gs_running_mremoteng.htm

```diff
@@ -0,0 +1,173 @@
+    <p>
+      Folders on mRemoteNG cannot only be used to categorize connections but also to
+      apply properties to the underlying connections.
+    </p>
+    <h2>Example</h2>
+    <p>
+      You have 10 Remote Desktop enabled servers in one domain and 15 in another domain.
+    </p>
+    <p>
+      Normally you would spend a lot of time creating all those connections and setting
+      the individual properties like username, password, etc. In mRemoteNG there is an
+      easier way. You just create two folders, one for domain A and one for domain B
+      and set all properties there.
+    </p>
+    <p>
+      Then create the Connections and let them inherit every property. The only
+      properties left to fill on Connection basis are the Connections name and hostname.
+      Everything else will be inherited from the parent folder.
+    </p>
+    <p>
+      Here is how you do this:
+    </p>
+    <h3>1. Add the folder</h3>
+    <p>
+      This can be done with:
+      <ul>
+        <li>Right click on connections and click on &nbsp;New Folder&nbsp;</li>
+        <li>&nbsp;File > New Folder&nbsp;</li>
+        <li>Or with keybinding: <kbd>Ctrl+Shift+N</kbd></li>
+      </ul>
+    </p>
+    <img src="Screenshots/Reference/add_new_folder.png">
+    <p>
+      Then give it a name and fill all the properties you need (like you did with
+      the test connection)
+    </p>
+    <img src="Screenshots/Reference/example_01.png"><br>
+    <img src="Screenshots/Reference/example_02.png">
+    <p>
+      When you have filled in the settings and values you can either just drag the test
+      Connection inside the folder or create a new one.
+    </p>
+    <img src="Screenshots/Reference/example_03.png">
+    <p>
+      Right now nothing has changed and nothing will be inherited.<br>
+      To enable inheritance switch to the inheritance view by clicking the
+      dedicated button. (Marked with a red arrow below)
+    </p>
+    <img src="Screenshots/Reference/example_04.png">
+    <p>
+      The properties that show up now are almost the same as before, but you
+      only select yes or no to enable or disable a inheritance.
+    </p>
+    <img src="Screenshots/Reference/example_05.png">
+    <p>
+      When no is selected the property will not be inherited, yes indicates
+      an inherited property.
+    </p>
+    <p>
+      For this test set &nbsp;Inherit Everything&nbsp; to Yes.
+    </p>
+    <p>
+      Now if you switch back to the properties view (the button left
+      of the inheritance button) you should see that not much is left of all
+      those properties.
+    </p>
+    <img src="Screenshots/Reference/example_06.png">
+    <p>
+      Only the Name and Hostname/IP properties are left over, everything else will be
+      inherited from the parent folder.
+    </p>
+    <p>
+      Of course you can also only let some of the properties be inherited. Just play
+      around with this a bit and you'll get the hang of it.
```

Comment:
This may also be a good time to show the Duplicate `Ctrl+D` shortcut to create several hosts within `My Domain A`. Go through setting up the `Test` connection and then show how you can create `Test2` and `Test3` records fairly easily.